warp/src/raygun/mod.rs

```diff
@@ -467,7 +467,7 @@ impl Conversation {
     }
 
     pub fn settings(&self) -> ConversationSettings {
-        self.settings
+        self.settings.clone()
```

Comment:
Not required right now but down the road we should try to borrow instead of cloning for the getters here. Would reduce allocations and let downstream handle how they wish to use the data (or if they wish to clone it).

Reply:
unchanged in the PR, but I agree 👍